db/column_family.cc

```diff
@@ -229,7 +229,19 @@ ColumnFamilyOptions SanitizeOptions(const ImmutableDBOptions& db_options,
   result.min_write_buffer_number_to_merge =
       std::min(result.min_write_buffer_number_to_merge,
                result.max_write_buffer_number - 1);
-  if (result.min_write_buffer_number_to_merge < 1) {
+  if (result.min_write_buffer_number_to_merge < 1 || db_options.atomic_flush) {
+    if (db_options.atomic_flush) {
+      ROCKS_LOG_WARN(
```

Comment:
Should we check for min_write_buffer_number_to_merge > 1 before this warning?

Reply:
Good catch. Will update.